guides/common/assembly_preparing-environment-for-installation.adoc

```diff
@@ -1,13 +1,13 @@
-[id="preparing-environment-for-satellite-installation"]
+[id="preparing-environment-for-installation"]
 = Preparing your Environment for Installation
 
```

Comment:
Fixed, I thought I had got them all.

Reply:
Thank you!

Reply:
Just a sidenote - we kept "satellite" in ids because this will ultimately break links downstream. We can probably create two `id=` statements one with satellite (legacy) and one newone if we want to be precise, but I am okay with this - people still land on the correct page, they just need to scroll. So I am good.